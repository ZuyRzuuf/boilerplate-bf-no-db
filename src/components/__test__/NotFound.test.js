import React from 'react';

import NotFound from '../NotFound';

describe('A suite', function() {
    test('renders isolated component correctly', () => {
        const wrapper = shallow(
            <NotFound />
        );
        expect(wrapper).toMatchSnapshot();
    });

    test('renders components tree correctly', () => {
        const wrapper = mount(
            <NotFound />
        );
        expect(wrapper).toMatchSnapshot();
    });

    test('renders HTML tree correctly', () => {
        const html = mount(
            <NotFound />
        );
        expect(html).toMatchSnapshot();
    });
});
