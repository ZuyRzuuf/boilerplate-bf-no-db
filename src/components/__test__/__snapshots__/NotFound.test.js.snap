// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A suite renders HTML tree correctly 1`] = `
<NotFound>
  <div>
    NotFound
  </div>
</NotFound>
`;

exports[`A suite renders components tree correctly 1`] = `
<NotFound>
  <div>
    NotFound
  </div>
</NotFound>
`;

exports[`A suite renders isolated component correctly 1`] = `
<div>
  NotFound
</div>
`;
